src/test/java/seedu/notus/command/ArchiveNoteCommandTest.java

```diff
@@ -40,6 +45,11 @@
     void setup() {
         notebook = new Notebook();
         storageManager = new StorageManager(timetable, parserManager, notebook, tagManager);
+        try {
+            storageManager.createFiles();
+        } catch (SystemException e) {
+            // not sure what to do here
+        }
```

Comment:
Since this test is on archive note, there is no need to create file. It just need a storageManager object.
```suggestion
```

Reply:
Its needed bc in the archive note command, the file gets deleted. So if there's no file instead of the actl message it'll return no file found

Reply:
Ah ok